base/util/src/main/java/com/netscape/cmsutil/json/JSONObject.java

```diff
@@ -0,0 +1,114 @@
+// --- BEGIN COPYRIGHT BLOCK ---
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; version 2 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+//
+// (C) 2021 Red Hat, Inc.
+// All rights reserved.
+// --- END COPYRIGHT BLOCK ---
```

Comment:
For new files we can use simpler license header like this:
https://github.com/dogtagpki/pki/blob/master/base/server/src/main/java/com/netscape/cmscore/apps/EngineConfig.java#L1-L5

Reply:
Fixed in next revision. I'll push it after you've taken a look at #3674 and it's merged in, otherwise the number of dependent PRs is going to get large!